### ISSUE TITLE
Development Topic - Accessibility

### ISSUE THREAD

**Issue by rossmurr4y-shine** [rossmurr4y-shine/work#117]
# Description

Loki Martin shared a link in Slack about an Accessability meetup called A11y Bytes. This got me thinking about accessibility in general, and how its something I know rather little about. I should do some research in this area and see how I might incorporate that into my self development plan.

## Acceptance Criteria

- [ ] Investigated accessibility self-development options 
- [ ] reached out to Loki and other advocates for some guidance in this area
- [ ] Created self-development tickets on my board

**Comment by rossmurr4y-shine** [comment]
The A11y videos are published here: https://www.youtube.com/@A11yBytes/videos
